recipes/r-gdalraster/meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set version = '2.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-gdalraster
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/gdalraster_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/gdalraster/gdalraster_{{ version }}.tar.gz
+  sha256: 5a52e24e10e44163dfc366007a356dfc27f9d28353d557b507e46d119f3b6ef2
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - r-rcpp
+    - r-rcppint64
+    - r-bit64
+    - r-nanoarrow
+    - r-wk
+    - r-xml2
+    - {{ compiler('c') }}              # [not win]
+    - {{ stdlib("c") }}                # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ stdlib("c") }}                # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
```

Comment:
```suggestion
    - {{ stdlib("c") }}
    - {{ compiler("cxx") }}
```

Unless there is something very weird with R on conda-forge compared to every other language this can just use the the standard compiler metapackages which know how to resolve correctly.

The CXX compilers should also get the required C compilers, so there shouldn't be a need for both, AFAIK.

cc @conda-forge/r for clarification on why the recipe generator is recommending this.